sodapy/__init__.py

```diff
@@ -229,6 +229,40 @@ def replace(self, dataset_identifier, payload, content_type="json"):
 
         return self._perform_update("put", resource, payload)
 
+    def createNonDataSet(self, payload, files, content_type="json"):
```

Comment:
Is it possible to upload multiple files at once? If not, `file` instead of `files`.